tensorflow_addons/utils/ensure_tf_install.py

```diff
@@ -17,27 +17,40 @@
 # Ensure TensorFlow is importable and its version is sufficiently recent. This
 # needs to happen before anything else, since the imports below will try to
 # import tensorflow, too.
+
+from distutils.version import LooseVersion
+import warnings
+
+import tensorflow as tf
+
+
+warning_template = """
```

Comment:
Just curious why this was pulled out of the function? Not likely to be an issue but if we don't need to add a variable to global space then I don't see a reason to?

Reply:
It's just easier to write as I don't have to break the indentation in the function to write the message. If you prefer that I put it in the function, I have nothing against that.